book/src/paper-wallet/installation.md

```diff
@@ -5,33 +5,37 @@ Follow this guide to setup Solana's key generation tool called `solana-keygen`
 After installation, ensure your version is `0.21.1` or higher by running `solana-keygen -V`
 {% endhint %}
 
-First, download the desired release tarball from GitHub. The examples below will
-retrieve the most recent release. If you would like to download a specific
-version instead replace `latest/download` with `download/VERSION` where VERSION
-is a tag name from https://github.com/solana-labs/solana/releases (ie. v0.21.1).
+## Download
+First, download the latest release tarball from GitHub.
 
-MacOS
-
-`$ curl -L -sSf -o solana-release.tar.bz2 'https://github.com/solana-labs/solana/releases/latest/download/solana-release-x86_64-apple-darwin.tar.bz2'`
-
-Linux
+**MacOS**
+```bash
+solana_downloads=https://github.com/solana-labs/solana/releases/latest/download
+solana_release=solana-release-x86_64-apple-darwin.tar.bz2
+curl -L -sSf -o solana-release.tar.bz2 $solana_downloads/$solana_release
+```
 
-`$ curl -L -sSf -o solana-release.tar.bz2 'https://github.com/solana-labs/solana/releases/latest/download/solana-release-x86_64-unknown-linux-gnu.tar.bz2'`
+**Linux**
+```bash
+solana_downloads=https://github.com/solana-labs/solana/releases/latest/download
+solana_release=solana-release-x86_64-unknown-linux-gnu.tar.bz2
+curl -L -sSf -o solana-release.tar.bz2 $solana_downloads/$solana_release
+```
 
+## Extract
 Next, extract the tarball
 ```bash
-$ tar xf solana-release.tar.bz2
+tar xf solana-release.tar.bz2
 ```
 
-Finally, `solana-keygen` can be run by
+## Add to "PATH"
+Now add the tool to your PATH environment variable with the following command
 ```bash
-$ solana-release/bin/solana-keygen
+export PATH="$(pwd)/solana-release/bin:${PATH}"
 ```
 
-If you would like to follow the remainder of these instructions without typing
-the leading path to `solana-keygen`, add it to your PATH environment variable
-with the following command
+## Check
+Finally, check that `solana-keygen` can be run by running
 ```bash
-$ export PATH="$(pwd)/solana-release/bin:${PATH}"
+solana-keygen
 ```
-This can be made permanent by adding it to your `~/.profile`
```

Comment:
@t-nelson I opted to remove this step. It's error-prone for a few reasons:
1. Doesn't work for `zsh` (https://superuser.com/a/187673)
> Zsh runs ~/.zprofile, not ~/.profile, when it is invoked as a login shell. The reason is that zsh has enough incompatibilities with standard shells to break scripts.
2. The `export PATH` line wouldn't work properly with `pwd`
3. A user would need to `source` the `.profile` to have it work in their current shell